app.js

```diff
@@ -188,6 +186,15 @@ app.post('/migrate', function (req, res) {
 
                         return jiraFetchIssues(project, firstIssue, lastIssue)
                             .then(function () {
+                                project.projects = Array.from(project.projects);
```

Comment:
Ah ok.   Given I think this is the last use of it (in terms of being a Set), that should be ok.

Otherwise perhaps another json-to-file library might support Set?